MobileWallet/Screens/Debug/UIViewControllerDebugExtension.swift

```diff
@@ -135,31 +103,116 @@ extension HomeViewController {
         present(alert, animated: true, completion: nil)
     }
 
-    override func motionBegan(_ motion: UIEvent.EventSubtype, with event: UIEvent?) {
-        let alert = UIAlertController(title: "Debug menu", message: nil, preferredStyle: .alert)
+    private func getEmailFooter() -> String {
+        var phoneDetails: [(key: String, value: String)] = []
 
-        alert.addAction(UIAlertAction(title: "View TariLib logs", style: .default, handler: { (_)in
-            self.showTariLibLogs()
-        }))
+        phoneDetails.append((key: "Network", value: TariSettings.shared.network.rawValue))
+        phoneDetails.append((key: "Phone", value: UIDevice.current.model.rawValue))
+        phoneDetails.append((key: "iOS", value: UIDevice.current.systemVersion))
 
-//        alert.addAction(UIAlertAction(title: "Generate test data", style: .default, handler: { (_)in
-//            self.generateTestData()
-//        }))
-//
-//        alert.addAction(UIAlertAction(title: "Simulate recieve transaction", style: .default, handler: { (_)in
-//            self.simulateReceieveTransactions()
-//        }))
+        if let version = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String {
+            phoneDetails.append((key: "App version", value: version))
+        }
 
-//        alert.addAction(UIAlertAction(title: "Simulate confirm sent transaction", style: .default, handler: { (_)in
-//            self.simulateConfirmSendTransactions()
-//        }))
+        if let build = Bundle.main.infoDictionary?["CFBundleVersion"] as? String {
+            phoneDetails.append((key: "App build", value: build))
+        }
 
-        alert.addAction(UIAlertAction(title: "Delete wallet", style: .destructive, handler: { (_)in
-            self.deleteWallet()
+        var footer = "<br/><br/><p><i>"
+        phoneDetails.forEach { (arg0) in
+            let (key, value) = arg0
+            footer.append("<br/>\(key): \(value)")
+        }
+        footer.append("</i></p>")
+
+        return footer
+    }
+
+    private func addAttachments(_ mail: MFMailComposeViewController) throws {
```

Comment:
Might save a lot of space if you zipped the files, per @CjS77's recommendation for Android.

Reply:
Good idea I'll do that thanks 👍